src/internal_events/amqp.rs

```diff
@@ -132,9 +106,9 @@ pub mod sink {
 
     impl InternalEvent for AmqpDeliveryError<'_> {
         fn emit(self) {
-            let reason = "Unable to deliver.";
+            let deliver_reason = "Unable to deliver.";
```

Comment:
I forgot where the conversation left of regarding the event-check script modifications. Did you happen to create an issue for that? I can if not.

Reply:
I have just raised it here #14409.

Reply:
Oh great, thanks!